content/akeneo-event-platform/concepts.md

```diff
@@ -185,45 +185,53 @@ To help identify duplicated events and deal with un-ordered events if it's somet
 
 Our delivery engine will try to deliver events as fast as possible but will adapt the throughput within the limits described in this section.
 
-Especially if you create subscriptions with HTTPS destination, ensure to respond `429 Too Many Requests` if your system is slightly overloaded: this way, the delivery engine will slow down quickly, retry undelivered events, and gently increase the throughput when your system gets back to normal (answers `200 OK` again).
+If your subscription has an HTTPS destination, please respond with `429 Too Many Requests` when your system is slightly overloaded: this way, the delivery engine will slow down, retry undelivered events, and gradually increase the throughput when your system gets back to recovers (i.e., when it responds `200 OK` again).
 
-Still, we will not retry indefinitely:
-- The platform will stop retrying to deliver events emitted more than 8 hours ago.
-- The platform will start to impact the failure rate, which can lead to a subscription suspension.
+The default delivery rate is 10 messages per second. Our system can increase the rate up to a maximum of 100 messages per second, or reduce it to as low as 1 message every 5 seconds. Please note that if the rate drops to 1 message every 5 seconds, the suspension policy will be triggered ([see bellow](/akeneo-event-platform/concepts.html#suspension-policy.html)).
```

Comment:
On explique vraiment tout ça en détail ? C'est des valeurs qui sont susceptibles de changer, pour moi, on dit juste qu'on va plus ou moins vite mais qui c'est trop lent c'est kapoutage de la subscription

Reply:
j'ai pas eu le temps de mettre un commentaire pour aborder ce point :D 
on peut se contenter de 
`Please note that if the delivery rate drops too significantly, the suspension policy will be triggered ([see bellow](/akeneo-event-platform/concepts.html#suspension-policy.html)).`

Reply:
Yup imo pas de valeur en message/s etc car en vrai on peu pas garantir que ça serra la cas ou qu'on va pas changer